climada_petals/entity/exposures/gdp_asset.py

```diff
@@ -86,7 +86,6 @@ def set_countries(self, countries=[], reg=[], ref_year=2000,
             self,
             data=Exposures.concat(gdp2a_list).gdf,
             ref_year=ref_year,
-            tag=Tag(description=description),
```

Comment:
The tag import in line 31 can be remove I think.

Reply:
Thanks for pointing this out!